### DISCUSSION TITLE
Prefill the cache during web service startup

### DISCUSSION THREAD

**Discussion by philipmvp** [SixLabors/ImageSharp.Web#370] [Q&A]
Is it possible to prefill the cache during web service startup using the files from the source directory. This is needed to improve the response time when called from the front end.

**Comment by JimBobSquarePants** [comment]
No. You need the full request path. 

I would suggest you profile your pages if you are having issues. Multi megapixel images can be processed and saved in milliseconds with ImageSharp.

**Reply by philipmvp** [reply]
Thanks for your quick reply. I will check if processing will be enough without a prefill.
